go.mod

```diff
@@ -11,6 +11,8 @@ replace github.com/denisenkom/go-mssqldb => github.com/grafana/go-mssqldb v0.0.0
 // It's also present on grafana/loki's go.mod so we'll need till it gets updated.
 replace k8s.io/client-go => k8s.io/client-go v0.18.8
 
+replace github.com/prometheus/alertmanager => github.com/gotjosh/alertmanager v0.21.1-0.20210329182206-9ab7cef7510e
```

Comment:
Hopefully this gets merged before this PR is reviewed and I don't have to use my own fork.